build.yaml

```diff
@@ -33,7 +33,10 @@ steps:
        valueFrom: default_ns.name
    public: false
    secrets:
+    - database-server-config
+    - gcr-push-service-account-key
     - test-gsa-key
+    - ci-gsa-key
```

Comment:
I'm not sure why this is necessary.  auth/create-accounts creates a (test) ci-gsa-key.

Reply:
I can't figure out why either. For some reason the ci-gsa-key is trying to be mounted in the first BuildImage step. I get this error: `MountVolume.SetUp failed for volume "gsa-key" : secret "ci-gsa-key" not found`. I'll keep investigating because it doesn't make sense why this is happening.

Reply:
It's in the batch1 code. We mount these volumes:

```
        volumes = [
            kube.client.V1Volume(
                secret=kube.client.V1SecretVolumeSource(
                    secret_name=self.userdata['gsa_key_secret_name']),
                name='gsa-key'),
            kube.client.V1Volume(
                secret=kube.client.V1SecretVolumeSource(
                    secret_name='batch-gsa-key'),
                name='batch-gsa-key')]

```

Reply:
So the "user" gsa key needs to be in the namespace for this to work!

Reply:
Oh, I see your concern. Let me think about this some more.

Reply:
Ok. I fixed it. We needed to also copy the ci-gsa-key in create-accounts to the batch-pods namespace.